index.html

```diff
@@ -0,0 +1,299 @@
+                            <div class="banner-article__title">DONEC NEC JUSTO EGET</div>
+                            <p lang="en" class="prh banner-article__prh">Morbi in sem quis dui placerat ornare. Pellentesque odio
+                            euismod in, pharetra a, ultricies in, diam.</p> 
+                        </div>
+                        <button class="btn btn--wide banner__btn">MORE INFO</button>
+                    </div>
+                </div>
+            </section>
+            <section class="service">
+                <ul class="service__list">
+                    <li class="service-item servise__list-item">
+                        <div class="circle-icon service-item__icon">
+                            <img class="service-item__icon-img" src="./resources/images/flag.svg">
+                        </div> 
+                        <div class="service-item__article">
+                            <h3 class="h3-like service-item__title">Ausce Cellentes</h3>
+                            <p lang="en" class="prh service-item__prh">Phasellus ultrices nulla quis nibh. Quisque a lectus. Donec consectetuer ligula vulputate sem tristique</p>
+                            <div class="service-item__more"><a class="link" href="#">Read More<span class="service-item__more-icon icon-arrow-circle-right"/></a></div>
+                        </div>
+                    </li>
+                    <li class="service-item servise__list-item">
+                        <div class="circle-icon service-item__icon">
+                            <img class="service-item__icon-img" src="./resources/images/camera.svg">
+                        </div> 
+                        <div class="service-item__article">
+                            <h3 class="h3-like service-item__title">Vestibul auctor</h3>
+                            <p lang="en" class="prh service-item__prh">Loasellus ultrices nulla quis nibh. Quisque a lectus. Donec consectetuer ligula vulputate sem tristique</p>
+                            <div class="service-item__more"><a class="link" href="#">Read More<span class="service-item__more-icon icon-arrow-circle-right"/></a></div>
+                        </div>
+                    </li>
+                    <li class="service-item servise__list-item">
+                        <div class="circle-icon service-item__icon">
+                            <img class="service-item__icon-img" src="./resources/images/home.svg">
+                        </div>
+                        <div class="service-item__article">
+                            <h3 class="h3-like service-item__title">Cras ornare</h3>
+                            <p lang="en" class="prh service-item__prh">Simasellus ultrices nulla quis nibh. Quisque a lectus. Donec consectetuer ligula vulputate sem tristique</p>
+                            <div class="service-item__more"><a class="link" href="#">Read More<span class="service-item__more-icon icon-arrow-circle-right"/></a></div>
```

Comment:
For better readability try to avoid long lines. Foe example move tag __a__ to a new line.